.gitignore

```diff
@@ -0,0 +1,2 @@
+*.pyc
```

Comment:
I'm not that interested. As it contains 'lib', which is a directory in this module, it has so far caused more problems than it has solved.